integrations/dynatrace/integration.py

```diff
@@ -0,0 +1,28 @@
+from port_ocean.core.handlers.port_app_config.api import APIPortAppConfig
+from port_ocean.core.handlers.port_app_config.models import (
+    PortAppConfig,
+    ResourceConfig,
+    Selector,
+)
+from port_ocean.core.integrations.base import BaseIntegration
+from pydantic.fields import Field
+
+
+class DynatraceEntitySelector(Selector):
+    entity_types: list[str] = Field(
+        default=["APPLICATION", "SERVICE"],
+        description="List of entity types to be fetched",
```

Comment:
Add and `alias = "entityTypes"`. this is what will be passed in the port-app-config when the user is specifying their interested entity types